<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBSDButton1Counters</class>
 <widget class="QMainWindow" name="WBSDButton1Counters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Button 1 Counters</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>639</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="23" column="0">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Volume counter (ml) FFLW Normal since start</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Volume counter (ml) Medium Strong since reset</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Volume counter (ml) Small Strong since start</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="rtdVol">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Volume counter (ml) Large Normal since reset</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="smallStrong">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="mediumStrong">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Volume counter (ml) Medium Mild since reset</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="smallStrongNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="largeNormal">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Volume counter (ml) RTD total since reset</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Volume counter (ml) Large Strong since start</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Volume counter (ml) Large Mild since start</string>
       </property>
      </widget>
     </item>
     <item row="24" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Volume counter (ml) FFLW Strong since start</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="smallMildNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="fflwNormal">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLabel" name="mediumMildNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="mediumNormal">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="fflwStrong">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="25" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Volume counter (ml) RTD total since start</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="fflwMild">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="smallNormalNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="largeMild">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QLabel" name="mediumStrongNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QLabel" name="largeMildNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="mediumMild">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QLabel" name="largeNormalNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLabel" name="mediumNormalNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="22" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Volume counter (ml) FFLW Mild since start</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Volume counter (ml) Medium Normal since reset</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Volume counter (ml) Medium Strong since start</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Volume counter (ml) Medium Normal since start</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Volume counter (ml) Large Normal since start</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Volume counter (ml) FFLW Normal since reset</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Volume counter (ml) FFLW Strong since reset</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Volume counter (ml) FFLW Mild since reset</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Volume counter (ml) Medium Mild since start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Volume counter (ml) Small Normal since reset</string>
       </property>
      </widget>
     </item>
     <item row="21" column="1">
      <widget class="QLabel" name="largeStrongNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Volume counter (ml) Small Normal since start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="smallNormal">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Volume counter (ml) Large Mild since reset</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Volume counter (ml) Large Strong since reset</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Volume counter (ml) Small Strong since reset</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="smallMild">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Volume counter (ml) Small Mild since reset</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="largeStrong">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Volume counter (ml) Small Mild since start</string>
       </property>
      </widget>
     </item>
     <item row="22" column="1">
      <widget class="QLabel" name="fflwMildNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="23" column="1">
      <widget class="QLabel" name="fflwNormalNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="24" column="1">
      <widget class="QLabel" name="fflwStrongNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="25" column="1">
      <widget class="QLabel" name="rtdVolNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBack">
    <property name="title">
     <string>Back</string>
    </property>
   </widget>
   <addaction name="menuBack"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
